designcompose/src/main/java/com/android/designcompose/TestUtils.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.android.designcompose
+
+import com.google.common.annotations.VisibleForTesting
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.flowOf
+
+/**
+ * For use in tests only
+ *
+ * Enable Live Update using a given Figma Token This skips the LiveUpdateSettings setup and
```

Comment:
Missing a period here. I'm confused about this and TestUtils.java. Are they supposed to be named the same?

Reply:
They don't need to be named the same. This file is specifically holding off a test only function that must be part of the package because it accesses internal variables. It can't be part of the TestUtils.java, because that is part of the new test fixture, and they're a seperate compile unit so they can't access internal variables